### DISCUSSION TITLE
Welcome to jobs.config Discussions!

### DISCUSSION THREAD

**Discussion by shoonia** [shoonia/jobs.config#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by joshbullough** [comment]
Can a job run more frequently than every hour?

**Reply by shoonia** [reply]
The document says that by using intervals as short 1 hour, the job will be ignored

> When using a cron expression, you can schedule your job to run at intervals as short as one hour apart, but not shorter. If you define your job to run more frequently, the job will be ignored.

[Velo: Scheduling Recurring Jobs](https://support.wix.com/en/article/velo-scheduling-recurring-jobs#cronexpression-string)
